content/handbook/product/ux/ux-forum.md

```
@@ -0,0 +1,165 @@
+    [低レベルの恥](/handbook/values/#low-level-of-shame-when-dogfooding)を目指し、できるだけ洗練度を低くして仕事を共有しましょう。
+    一部のトピックは、より洗練されたストーリーから利益を得ますが、この洗練は品質の認識を高め、他のスピーカーに不必要なプレッシャーをかけ、他の人が関与するのを難しくする可能性があります。
+- **共感できる**:
+    オーディエンスを念頭に置いてストーリーを構成し、彼らからフィードバックを引き出し、潜在的なオーバーラップをハイライトします。
+    これにより、人々が関連付けたり、価値を見つけたり、関与したりしやすくなります。
+    たとえば、意図的にオーバーラップを指摘したり、オーディエンスへの質問を追加したりします。
+- **イテレーションマインドセット**:
+    他の人から学び、協力する機会として使いましょう。
+    これまでにやった仕事だけでなく、どのようにイテレートしていくか、何が欠けているかについてもイテレーションを輝かせましょう。
+    たとえば、現在の課題、未解決の質問、他の人がどこで助けられるかをリストアップします。
+
+[役立つヒント](#helpful-tips-for-those-sharing)を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)